frappe/website/page_renderers/static_page.py

```diff
@@ -8,7 +8,7 @@
 from frappe.website.page_renderers.base_renderer import BaseRenderer
 from frappe.website.utils import is_binary_file
 
-UNSUPPORTED_STATIC_PAGE_TYPES = ("html", "md", "js", "xml", "css", "txt", "py", "json")
+UNSUPPORTED_STATIC_PAGE_TYPES = ("html", "md", "js", "xml", "css", "txt", "py", "pyc", "json")
```

Comment:
```suggestion
UNSUPPORTED_STATIC_PAGE_TYPES = ("html", "md", "js", "xml", "css", "txt", "py", "pyc", "json", "pyo")
```

if started with `-o` flag pyo are created instead of pyc.